Source/WebCore/rendering/svg/RenderSVGText.cpp

```diff
@@ -324,13 +324,6 @@ void RenderSVGText::layout()
     // textElement().updateLengthContext();
 
     bool updateCachedBoundariesInParents = false;
-    if (!isLayerBasedSVGEngineEnabled()) {
```

Comment:
LBSE no longer makes use of the "m_localTransform" here, so you can avoid the computation in the section where you move the code to if LBSE is on. Note: in LBSE currently transformations on <text> are ignored, your patch most likely fixes that too :-)